robot/pmm/resources/pmm.py

```diff
@@ -25,10 +31,37 @@ def __init__(self, debug=False):
         self.debug = debug
         self.current_page = None
         self._session_records = []
+        self.val=0
+        self.payment_list= []
         # Turn off info logging of all http requests
-        logging.getLogger("requests.packages.urllib3.connectionpool").setLevel(
-            logging.WARN
-        )
+        logging.getLogger('requests.packages.urllib3.connectionpool').setLevel(logging.WARN)
+        self._init_locators()
+        # patch salesforce locators for winter 21
+        if int(self.latest_api_version) == 50:
+            from cumulusci.robotframework import Salesforce
+            Salesforce.lex_locators["record"]["related"]["card"] = (
+                "//a[contains(@class,'slds-card__header-link')][.//span[@title='{}']]"
+            )
+            Salesforce.lex_locators["record"]["related"]["button"]=("//article[contains(@class, 'slds-card slds-card_boundary')][.//span[@title='{}']]//a[@title='{}']")
+            Salesforce.lex_locators["record"]["related"]["popup_trigger"]=("//article[.//span[@title='{}'][//a[text()='{}']]]//div[contains(@class,'slds-truncate')]//button[./span[text()='Show Actions']]")
```

Comment:
u dont need the salesforce locators patch anymore, bryan fixed those locators in latest cci upgrade

Reply:
Removed the patch